docs/tutorials/tut-minemeld-migration.md

```diff
@@ -0,0 +1,291 @@
+
+You can output indicators with Cortex XSOAR by using two integrations, _Palo Alto Networks PAN-OS EDL Service_ and _Export Indicators Service_. Migrating MineMeld output nodes to Cortex XSOAR is a process that requires looking at the prototype of a given output node, as well as the prototypes of all of the nodes that flow into that output node. We need to do this to understand how to construct the query we will enter when configuring an instance of the _Palo Alto Networks PAN-OS EDL Service_ or _Export Indicators Service_ integration.
+
+A concrete example will demonstrate how this is done. Here is an example output node in MineMeld.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/feed-hc-green-output-node.png)
+
+The first step is to look at the output node's prototype, which we can do by clicking the link `stdlib.feedHCGreenWithValue` as seen in the previous screenshot. It appears as follows.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/feed-hc-green-output-prototype.png)
+
+We see in the `config` section that this prototype filters for indicators whose `confidence` is greater than 75 and whose `share_level` is 'green' - this is the first bit of information we need.
+Now let's go back to the node's inputs. We need to explore each one. Our example only has one input node listed - let's explore it by clicking the link there `aggregatorIPv4Inbound-clone-MLB`.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/ipv4-aggregator-node.png)
+
+Let's see the details of the node's prototype by clicking the prototype linked there - `stdlib.aggregatorIPv4Inbound`.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/ipv4-aggregator-prototype.png)
+
+We see in the `config` section that this prototype filters for indicators whose `type` is 'IPv4' - let's file this information away for when we configure an integration instance in Cortex XSOAR.
+Now if we go back to the aggregator node, we see that it too only has one input node listed. Let's perform the same actions as before and we'll be finished gathering the information we need. Let's click on the input node linked as `exit_addresses-clone-MLB`.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/exit_addresses-node.png)
+
+Click on the prototype listed there to see more details" `tor.exit_addresses`.
+
+We know from the table mapping MineMeld prototypes to Cortex XSOAR integrations detailed [above](#Minemeld-Prototype-to-Cortex-XSOAR-Integration-Mapping) that the `tor.exit_addresses` prototype maps to the _Tor Exit Addresses Feed_ integration. And with that, we have all the information we need to move forward. Let's review and gather here the information we collected from looking at the output node we want to migrate, and all the nodes that flow into it.
+1. Filters for indicators whose `confidence` is greater than 75 and whose `share_level` is 'green'
+2. Filters for indicators whose `type` is 'IPv4'
+3. The `tor.exit_addresses` prototype maps to the _Tor Exit Addresses Feed_ integration
```

Comment:
The list is not displaying correctly in Netlify, please add an empty line before and after